.gitignore

```diff
@@ -1,10 +1,10 @@
-# Generated binary files
-/obj/
-/dnsmasq/obj
+# Object files output by the assembler
+/build
+
+# Generated binary
 pihole-FTL
-socket-test
```

Comment:
This binary has been removed from our Makefile some time ago. Its source code is conserved under `/test/` but is not used by default.